src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.cs

```diff
@@ -209,7 +209,14 @@ public override void VisitLocal(ILocalSymbol symbol)
                 AddSpace();
             }
 
-            builder.Add(CreatePart(SymbolDisplayPartKind.LocalName, symbol, symbol.Name));
+            if (symbol.IsConst)
+            {
+                builder.Add(CreatePart(SymbolDisplayPartKind.ConstantName, symbol, symbol.Name));
```

Comment:
should we have different part kinds for local-constants versus field constants?  i'm open to either approach.

Reply:
However, i lean toward treating these in distinct fashion.

Reply:
We decided in the design meeting to classify these both as constant for the time being. Field constants will additionally be tagged with the static symbol classification.